### PR TITLE
(maint) Add /opt/puppetlabs/bin to path

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: configs/components/razor-server.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 component "razor-server" do |pkg, settings, platform|
   pkg.load_from_json('configs/components/razor-server.json')
+  pkg.add_source "file://resources/files/razor-servr.sh", sum: "f5987a68adf3844ca15ba53813ad6f63"
```
Comment (mckern) [inline configs/components/razor-server.rb]: typo: `razor-servr.sh`

**Comment by smcclellan** [comment]
👍

**Comment by melissa** [comment]
Oops, good catch Ryan. So, changed that, and this builds successfully.

File: resources/files/razor-server.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Add /opt/puppetlabs/bin to the path for sh compatible users
```
Comment (mckern) [inline resources/files/razor-server.sh]: You had asked if we care about C shells, and the answer is "I don't know, but we can probably replicate this for C shells too if you want to"

**Pull request merged**